### DISCUSSION TITLE
System monitoring can't be done with New Relic, so what should we use?

### DISCUSSION THREAD

**Discussion by geoidesic** [php/frankenphp#1064] [Q&A]
Is there a recommendation for monitoring system events? My initial plan was to use New Relic but apparently that's not supported: https://frankenphp.dev/docs/known-issues/#unsupported-php-extensions

What is the recommendation?

My goal is to be able to see:

- CPU / memory usage graphs
- Requests per second graphs
- Error notifications to something like Slack

**Comment by dunglas** [comment]
We expose Prometheus metrics and structured log. You can collect them with Prometheus, an ELK stack, DataDog and many other solutions.

- https://caddyserver.com/docs/metrics
- https://caddyserver.com/docs/logging
